src/west/_bootstrap/main.py

```diff
@@ -115,26 +115,59 @@ def init(argv):
     This exits the program with a nonzero exit code if fatal errors occur.'''
     init_parser = argparse.ArgumentParser(
         prog='west init',
-        description='Bootstrap initialize a Zephyr installation')
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        description=
+'''
+Initializes a West instance.
```

Comment:
What do you think about "Initializes a Zephyr installation."?

I think it's important to keep the focus on this being the command to install *zephyr*, but using west.

Reply:
Alright, I'll change it back.

Reply:
Done